src/sdk/model/company/support-ticket/support-regarding-option.ts

```diff
@@ -0,0 +1,51 @@
+import { SupportRegardingOptionJson, SupportServiceOptionJson } from './__json__';
+
+/**
+ * Support Regarding Option.
+ */
+/* istanbul ignore next: autogenerated */
+export class SupportRegardingOption {
+
+  constructor(private _json: SupportRegardingOptionJson) {
+  }
+
+  /**
+   * Get id.
+   * @returns {number}
+   */
+  get id(): number {
+    return this._json.id;
+  }
+
+  /**
+   * Get name.
+   * @returns {string}
+   */
+  get name(): string {
+    return this._json.name;
+  }
+
+  /**
+   * Get service options.
+   * @returns {Array<SupportServiceOptionJson>}
+   */
+  get serviceOptions(): Array<SupportServiceOptionJson> {
```

Comment:
Needs to return `Array<SupportServiceOption>`

Reply:
this is back to returning the Json interface?
you should have `return this.json.service_options.map(it => new SupportServiceOption(it));`